tasks/agent6-linux.yml

```diff
@@ -58,14 +58,29 @@
     group: "{{ datadog_group }}"
   notify: restart datadog-agent
 
+- name: Create system-probe configuration file
+  template:
+    src: system-probe.yaml.j2
+    dest: /etc/datadog-agent/system-probe.yaml
+    mode: 0644
+    owner: "root"
+    group: "root"
+
 - name: Ensure datadog-agent is running
   service:
     name: datadog-agent
     state: started
     enabled: yes
   when: not datadog_skip_running_check and datadog_enabled and not ansible_check_mode
 
-- name: Ensure datadog-agent, datadog-agent-process and datadog-agent-trace are not running
+- name: Ensure datadog-agent-sysprobe is running if enabled
```

Comment:
why are we checking twice that the system-probl is is running ? Since it's added to the list line 93

Reply:
Removed